stubs/passlib/@tests/stubtest_allowlist.txt

```diff
@@ -90,3 +90,8 @@ passlib.utils.compat.*
 
 # Tests are not included:
 passlib.tests.*
+
+# This is only available when the crypt module is available. This module
+# was dropped from the standard library of Python 3.13, but is still available
+# in some environments.
```

Comment:
I haven't found out which package installs `crypt.py`, but I have had it happen once locally, and the same issue seems to happen in CI. (In this particular run. Previously, it complained that `passlib.hosts.host_context` was spurious. 🤷‍♂️)